src/components/Notification.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <Transition
+    appear
+    name="fade"
+  >
+    <div class="notification">
+      <div class="content">
+        <slot>{{ text }}</slot>
+      </div>
+
+      <footer v-if="$slots.footer">
+        <slot name="footer" />
+      </footer>
+    </div>
+  </Transition>
+</template>
+
+<script>
+export default {
+  props: {
+    resolve: { type: Function, required: true },
+    text: { type: String, default: '' },
+  },
+  mounted() {
+    setInterval(() => this.resolve(), 5000);
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+@import '~@aeternity/aepp-components-3/src/styles/placeholders/typography.scss';
+@import '~@aeternity/aepp-components-3/src/styles/variables/colors.scss';
+
+.notification {
+  position: fixed;
+  top: 0;
+  left: rem(10px);
+  right: rem(10px);
+  padding-top: env(safe-area-inset-top);
+  border-radius: 0 0 rem(8px) rem(8px);
+  background-color: $color-neutral-maximum;
+  box-shadow: 0 0 rem(100px) rem(30px) rgba(146, 156, 166, 0.4);
+  @extend %face-sans-base;
+
+  &.fade-enter-active, &.fade-leave-active {
+    transition: opacity 0.25s ease-out;
+  }
+
+  &.fade-enter, &.fade-leave-to {
+    opacity: 0;
+  }
+
+  .content {
+    overflow-wrap: break-word;
```

Comment:
We were using this before:
```
overflow: hidden;
overflow-wrap: break-word;
```
I can't prove that `overflow: hidden` is needed in this case.